sklearn/feature_extraction/image.py

```diff
@@ -503,17 +503,19 @@ class PatchExtractor(TransformerMixin, BaseEstimator):
     Parameters
     ----------
     patch_size : tuple of int (patch_height, patch_width), default=None
-        The dimensions of one patch.
+        The dimensions of one patch. If set to None, the patch size will be
+        automatically set to (img_height // 10, img_width // 10), where
+        img_height and img_width are the dimensions of the input images.
 
     max_patches : int or float, default=None
         The maximum number of patches per image to extract. If `max_patches` is
         a float in (0, 1), it is taken to mean a proportion of the total number
-        of patches.
+        of patches. If set to None, extract all possible patches.
 
     random_state : int, RandomState instance, default=None
         Determines the random number generator used for random sampling when
         `max_patches is not None`. Use an int to make the randomness
-        deterministic.
+        deterministic. If set to None, use the RandomState from np.random.
```

Comment:
I would revert this change. `random_state` is a specific one because we define more precisely in the Glossary. Currently, all entry of `random_state` should be consistent in the doc (sometime missing what is the process is random).

Reply:
Updated to incorporate your changes.